pkg/controllers/externalsecret/externalsecret_controller.go

```diff
@@ -128,7 +130,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		conditionSynced := NewExternalSecretCondition(esv1beta1.ExternalSecretReady, v1.ConditionFalse, esv1beta1.ConditionReasonSecretSyncedError, errStoreRef)
 		SetExternalSecretCondition(&externalSecret, *conditionSynced)
 		syncCallsError.With(syncCallsMetricLabels).Inc()
-		return ctrl.Result{RequeueAfter: requeueAfter}, nil
+		return ctrl.Result{}, err
```

Comment:
Well... I forgot to remove this extra. If the secretStore is not already created... well should retry quickly. Will prevent some issue we met on slow kubeapi with a fixed 30sec retry)